src/ngraph/graph_util.hpp

```diff
@@ -96,17 +96,11 @@ namespace ngraph
             result_list.push_back(node_map[independent_node]);
             independent_nodes.pop_front();
 
-            for (auto& output : independent_node->get_outputs())
+            for (auto user : independent_node->get_users())
```

Comment:
I call auto abuse here. Use `auto&` or better yet `const shared_ptr<Node>&`.  Topological sort is called frequently and creating temporaries can be expensive. Same for line 176